spec/features/find/v2/results/search_results_feature_flag_disabled_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'V2 results - feature flag disabled' do
+  before do
+    allow(Settings.features).to receive_messages(v2_results: false)
+  end
+
+  scenario 'when I visit the results page' do
+    when_i_visit_the_find_results_page
+    then_i_see_not_found
+  end
+
+  def when_i_visit_the_find_results_page
+    visit find_v2_results_path
+  end
+
+  def then_i_see_not_found
+    expect(page.status_code).to eq(404)
```

Comment:
The feature tests sounds like a request spec but just for this PR. 

Next piece of work on this would be the actual behaviour expected and this step will be removed.